src/utils/string_parsing.ts

```diff
@@ -69,6 +81,17 @@ function utf16LEToStr(bytes : Uint8Array) : string {
  * @returns {string}
  */
 function beUtf16ToStr(bytes : Uint8Array) : string {
+  if (typeof window.TextDecoder === "function") {
```

Comment:
Maybe this check can be done once.

Reply:
I hesitated between the two but ended with doing that instead for multiple reasons:
  1. If we move it back to the global scope, we would also have to use `isNode` as a condition
  2. This is not a performance sensitive operation nor a a performance-sensitive place in the code
  3. I find it more readable that way
  4. The TextDecoder can be polyfilled after the RxPlayer is imported that way. Though I don't know if that's a real use case

Reply:
Ok, I understand the arguments. I was especially thinking about the performance, but you're right, it is not really a performance-sensitive operation, it may be better readable than over-optimized.